README.md

```diff
@@ -28,6 +28,37 @@ curl -L https://github.com/Cox-Automotive/terraform-provider-alks/releases/downl
 
 * Note: If you've used a previous version of the ALKS provider and created a `.terraformrc` file in your home directory you'll want to remove it prior to updating.
 
+## Terraform version >= 0.13 Terraform Installation
+
+* Download and install [Terraform](https://www.terraform.io/intro/getting-started/install.html)
+
+* Download ALKS Provider binary for your platform from [Releases](https://github.com/Cox-Automotive/terraform-provider-alks/releases)
+  
+For example on macOS:
+
+```
+curl -L https://github.com/Cox-Automotive/terraform-provider-alks/releases/download/1.4.3/terraform-provider-alks-darwin-amd64.tar.gz | tar zxv
+```
+
+* Go into the Terraform plugins path; `.terraform.d/plugins/` on MacOS/Linux or `terraform.d\plugins\` in your user's "Application Data" directory on Windows.
+
+* Create the following directories: `coxautoinc.com/engineering-enablement/alks/1.4.3/darwin_amd64` and put the binary into the `darwin_amd64/` directory.
```

Comment:
Let's add a small note here about OS and architecture.  I know that the example uses MacOS x86-64, but we also note things about Windows installation in the previous step.  Like perhaps the path should read more like `coxautoinc.com/engineering-enablement/alks/1.4.3/<OS>_<ARCH>` and give an example for 64bit MacOS (`darwin_amd64`) and a separate one for windows (`windows_amd64`).

Reply:
Ah good call here, I've updated and added examples as you suggested 👍